### DISCUSSION TITLE
unable to install postgresql drivers in Dbeaver

### DISCUSSION THREAD

**Discussion by imsandeepjindal** [dbeaver/dbeaver#17378] [Q&A]
I am trying to connect a postgresql database and whenever i download the drivers, it shows postrgesql drivers are missing.
when i connect, it throws an error: connection time out
![db1](https://user-images.githubusercontent.com/58435200/185174617-7d739d3b-0914-4a4e-a39b-af1d7b178b74.PNG)
![db2](https://user-images.githubusercontent.com/58435200/185174628-133e9bab-f1b8-48c8-ae92-360e506635a7.PNG)
![db3](https://user-images.githubusercontent.com/58435200/185174629-5bd01923-8622-4d0a-986e-ea9be8efce4a.PNG)

using port:5444

**Comment by thesarangsaraf** [comment]
you can manually download postgres jdbc drivers from website and hard link it.
https://jdbc.postgresql.org/download.html

**Comment by imsandeepjindal** [comment]
I tried it still the same case, if you can help me with the steps to hard
link it that would be great
